apps/api/v2/test/fixtures/oauth-client.repository.fixture.ts

```diff
@@ -0,0 +1,37 @@
+import { CreateOAuthClientInput } from "@/modules/oauth/input/create-oauth-client";
```

Comment:
I called this "OAuthClientRepositoryFixture" instead of "OAuthClientFixture" just like other fixtures, because during the test I need to instantiate OAuthClient:

let oauthClient: PlatformOAuthClient;

and if we would call fixture "PlatformOAuthClientFixture" then it is confusing.

Reply:
@ThyMinimalDev  thoughts?

Reply:
not sure to see the issue with naming the fixture PlatformOAuthClientFixture

Reply:
I would just name it like the other fixtures, and import with alias if needed

"import OAuthClientFixture as OAuthClientRepositoryFixture from ..."

Reply:
But why are we not calling them repository fixtures if they perform CRUD operations just like real repositories?

Reply:
good point, if it makes more sense we can rename all the other fixtures

Reply:
Done in [refactor: test repository fixtures](https://github.com/calcom/cal.com/pull/12814/commits/665f1d68c0e093aef7502cbd4923f5ab86dd6845).

confirmed that "yarn test" and "yarn test:e2e" work.